test/lib/mayaUsd/render/mayaToHydra/cpp/testFlowViewportAPIViewportInformation.py

```diff
@@ -25,6 +25,10 @@ class TestFlowViewportAPIViewportInformation(mtohUtils.MayaHydraBaseTestCase):
     _file = __file__
 
     def test_RendererSwitching(self):
+
+        # reset all default viewports for VP2 as the test framework sets it to Storm in base class    
+        self.setViewport2Renderer()
```

Comment:
I don't think that is useful to set that line since line 34 we set Storm as the renderer